labs/lab3/readme.md

```diff
@@ -50,9 +50,11 @@ Azure Kubernetes Service is a service provided for Kubernetes on Azure infrastru
 
 ### Deploy first Kubernetes Cluster with Azure CLI
 
-1. With the use of single Azure CLI command, you will deploy a production-ready AKS cluster with some additional options. (**This will take a while**).
+![AKS Icon](media/aks-icon.png)
 
-   First, initialise the Environment variables based on your setup, which are passed to the Azure CLI command as shown below:
+With the use of single Azure CLI command, you will deploy a production-ready AKS cluster with some additional options. (**This will take a while**).
+
+1. First, initialise the Environment variables based on your setup, which are passed to the Azure CLI command as shown below:
```

Comment:
_initialise_ is the international spelling of the word.  _initialize_ is the US version.  Both are right, so change it or don't,  just an FYI